cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -5819,6 +5833,191 @@ subroutine init_snowtable
 
       end subroutine init_snowtable
 
+!=======================================================================
+
+! snicar snow grain single-scattering properties lookup table
+!
+!
+     subroutine init_snicarssptable
+
+      use ice_broadcast, only: broadcast_array, broadcast_scalar
+      integer (kind=int_kind) :: &
+         nsnwgrad, &   ! Table dimensions
+         nspecint
```

Comment:
nswgrad = nmbrad in icepack_shortwave.  I'd prefer nsnwradii, to reduce confusion about what 'rad' (or 'grad') refers to.
nspecint = nspint in icepack_shortwave, and isn't restricted to snow there.  So here, maybe nsnwspint?

snicar is okay in this subroutine name since it's the data source

Reply:
Will do.